docs/source/include/learning.rst

```diff
@@ -0,0 +1,93 @@
+Learned Models
+==============
+
+In SCICO, neural network models are used to represent imaging problems and provide different modes of data-driven regularization.
+The models are implemented in `Flax <https://flax.readthedocs.io/>`_, and constitute a representative sample of frequently used networks.
+
+
+FlaxMap
+-------
+
+SCICO interfaces with the implemented models via :class:`.FlaxMap`. This provides a standardized access to all trained models via the model definiton and the learned parameters. Further specialized functionality, such as learned denoisers, are built on top of :class:`.FlaxMap`. The specific models that have been implemented are described below.
+
+
+
+DnCNN
+-----
+
+The denoiser convolutional neural network model (DnCNN) :cite:`zhang-2017-dncnn`, implemented as :class:`.DnCNNNet`, is used to denoise images that have been corrupted with additive Gaussian noise.
+
+
+
+ODP
+---
+
+The unrolled optimization with deep priors (ODP) :cite:`diamond-2018-odp`, implemented as :class:`.ODPNet`, is used to solve inverse problems in imaging by adapting classical iterative methods into an end-to-end framework that incorporates deep networks as well as knowledge of the image formation model.
+
+The framework aims to solve the optimization problem
+
+    .. math::
+       \argmin_{\mb{x}} \; f(A \mb{x}, \mb{y}) + r(\mb{x}) \;,
+
+where :math:`A` represents a linear forward model and :math:`r` a regularization function encoding prior information, by unrolling the iterative solution method into a network where each iteration corresponds to a different stage in the ODP network.
+
+
+Unrolled Optimization Algorithms
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Different iterative solution algorithms produce different ODP networks. The ones implemented in SCICO are
+
+* Proximal Map: This algorithm corresponds to solving
+
+    .. math::
+       :label: eq:odp_prox
+
+       \argmin_{\mb{x}} \; \alpha_k \, f(A \mb{x}, \mb{y}) + \frac{1}{2} \| \mb{x} - \mb{x}^k - \mb{x}^{k+1/2} \|_2^2 \;,
+
+  with :math:`k` corresponding to the index of the iteration, which translates to an index of the stage of the network, :math:`f(A \mb{x}, \mb{y})` a fidelity term, usually an :math:`\ell_2` norm, and :math:`\mb{x}^{k+1/2}` a regularization representing :math:`\mathrm{prox}_r (\mb{x}^k)` and usually implemented as a convolutional neural network (CNN). This proximal map representation is used when minimization problem :eq:`eq:odp_prox` can be solved in a computationally efficient manner.
+
+  :class:`.ODPProxDnBlock` uses this formulation to solve a denoising problem, yielding
+
```

Comment:
Is there a reference that could be added that provides further detail on this equation?

Reply:
The original reference. Included again for rapid referencing.